src/OmniSharp.Http.Driver/OmniSharp.Http.Driver.csproj

```diff
@@ -11,6 +11,7 @@
         <!-- Always run on the latest runtime installed. -->
         <RuntimeFrameworkVersion>6.0.0-preview.7.21317.1</RuntimeFrameworkVersion>
         <RollForward>LatestMajor</RollForward>
+        <SkipHostDlls>true</SkipHostDlls>
```

Comment:
We weren't removing these libraries from the Full Framework builds. Should this be conditioned by TFM?